privacy-protections/running-tests.md

```diff
@@ -0,0 +1,14 @@
+# Running tests
+
```

Comment:
Do we have a public glitch link we can put in here? It might also make sense to document the glitch deployment and change process here at some point.

Reply:
Good idea - I added a document with deployment information: https://github.com/duckduckgo/privacy-test-pages/pull/11/commits/b98b353f2d710a3051d4355765ebd20c462990f8 (I plan to make glitch automatic later on). I also added a note about glitch to this doc.